### PR TITLE
May 2021 relnotes.md

### PR DESCRIPTION
**Pull request by KittyYeungQ**

### CODE REVIEW

File: articles/qdk-relnotes.md [hunk 1]
```diff
@@ -20,6 +20,27 @@ For installation instructions, please refer to the [install guide](xref:microsof
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update-qdk).
 
 
+*Release date: May 24th, 2021*
+
+## QDK Version 0.17.2105.xxxxxx
+
+- Updated control-plane clients (.NET, Go, Python) with new swagger. See full details in [Azure REST API specs](https://github.com/Azure/azure-rest-api-specs/pull/13002).
+- Added a new function to the `azure-quantum` Python client to support the translation of binary optimization terms from `npz` to Azure Quantum. See full details in [QDK Python](https://github.com/microsoft/qdk-python/pull/61).
+- Published QIR oracle compiler sample. This program allows turns classical Q# functions on Boolean inputs into quantum implementations of that functions in terms of Q# operations at the level of QIR. This allows, for example, to implement quantum algorithms that are used by many quantum algorithms readily as classical functions.
```
Comment (msoeken) [inline articles/qdk-relnotes.md:29]:
```suggestion
- Published [QIR oracle generation sample](https://github.com/microsoft/Quantum/issues/496). This program allows turns classical Q# functions on Boolean inputs into quantum implementations of that functions in terms of Q# operations at the level of QIR. This allows, for example, to implement quantum algorithms that are used by many quantum algorithms readily as classical functions.
```

**Review by frtibble — APPROVED** [review]

File: articles/qdk-relnotes.md [hunk 2]
```diff
@@ -20,6 +20,27 @@ For installation instructions, please refer to the [install guide](xref:microsof
...
+- Fixed a bug that prevents QIR generation from being enabled in the iqsharp-base Docker image. See details [here](https://github.com/microsoft/iqsharp/issues/432).
+- Implemented special functions proposal.
+- C# Client: Changed input data format type to "v2" for Quantum Computing. 
+- Set CLI user agent to include version number.
+
+## Azure Quantum update
+
+- Job Scheduler: Fixed an issue where jobs could be rejected if an unrelated metric has reached capacity.
+- Control Plane: Set Provider in Failed state if provisioning fails. Previously it would be stuck in Launching/Updating state.
+- Added help button to direct user to support forum.
```
Comment (xinyi-joffre) [inline articles/qdk-relnotes.md:40]: Added help button in portal to direct user to support forum.

File: articles/qdk-relnotes.md [hunk 3]
```diff
@@ -20,6 +20,27 @@ For installation instructions, please refer to the [install guide](xref:microsof
...
+- Rendered provider cost in localized currency from Azure Marketplace.
```
Comment (xinyi-joffre) [inline articles/qdk-relnotes.md:41]: Added feedback button in portal to gather user feedback.

File: articles/qdk-relnotes.md [hunk 4]
```diff
@@ -20,6 +20,27 @@ For installation instructions, please refer to the [install guide](xref:microsof
...
+
```
Comment (xinyi-joffre) [inline articles/qdk-relnotes.md:42]: Added quickstart guide in portal in overview blade.

File: articles/qdk-relnotes.md [hunk 5]
```diff
@@ -20,6 +20,27 @@ For installation instructions, please refer to the [install guide](xref:microsof
...
+- Added a new function to the `azure-quantum` Python client to support the translation of binary optimization terms from `npz` to Azure Quantum. See full details in [QDK Python](https://github.com/microsoft/qdk-python/pull/61).
+- Published [QIR oracle generation sample](https://github.com/microsoft/Quantum/issues/496). This program allows turns classical Q# functions on Boolean inputs into quantum implementations of that functions in terms of Q# operations at the level of QIR. This allows, for example, to implement quantum algorithms that are used by many quantum algorithms readily as classical functions.
+- Added Chemistry azure-quantum sample notebook to Quantum repo. 
+- Fixed a bug that prevents QIR generation from being enabled in the iqsharp-base Docker image. See details [here](https://github.com/microsoft/iqsharp/issues/432).
+- Implemented special functions proposal.
+- C# Client: Changed input data format type to "v2" for Quantum Computing. 
+- Set CLI user agent to include version number.
+
+## Azure Quantum update
+
```
[the remark this answers is not in the document]
Comment (efratshabtai) [reply articles/qdk-relnotes.md:37]: This is a service side change, when was this deployed? It should be another entry in the release notes. (not part of the QDK)

File: articles/qdk-relnotes.md [hunk 6]
```diff
@@ -20,6 +20,27 @@ For installation instructions, please refer to the [install guide](xref:microsof
...
+*Release date: May 24th, 2021*
+
+## QDK Version 0.17.2105.xxxxxx
+
+- Updated control-plane clients (.NET, Go, Python) with new swagger. See full details in [Azure REST API specs](https://github.com/Azure/azure-rest-api-specs/pull/13002).
+- Added a new function to the `azure-quantum` Python client to support the translation of binary optimization terms from `npz` to Azure Quantum. See full details in [QDK Python](https://github.com/microsoft/qdk-python/pull/61).
+- Published [QIR oracle generation sample](https://github.com/microsoft/Quantum/issues/496). This program allows turns classical Q# functions on Boolean inputs into quantum implementations of that functions in terms of Q# operations at the level of QIR. This allows, for example, to implement quantum algorithms that are used by many quantum algorithms readily as classical functions.
+- Added Chemistry azure-quantum sample notebook to Quantum repo. 
+- Fixed a bug that prevents QIR generation from being enabled in the iqsharp-base Docker image. See details [here](https://github.com/microsoft/iqsharp/issues/432).
+- Implemented special functions proposal.
```
Comment (efratshabtai) [inline articles/qdk-relnotes.md:32]: Can we add a link to the proposal and PR?

File: articles/qdk-relnotes.md [hunk 7]
```diff
@@ -20,6 +20,27 @@ For installation instructions, please refer to the [install guide](xref:microsof
...
+- C# Client: Changed input data format type to "v2" for Quantum Computing. 
+- Set CLI user agent to include version number.
+
+## Azure Quantum update
```
Comment (efratshabtai) [inline articles/qdk-relnotes.md:36]: The service deployed every week - when were these updated?
Do we add a date to each service update?

File: articles/qdk-relnotes.md [hunk 8]
```diff
@@ -20,6 +20,27 @@ For installation instructions, please refer to the [install guide](xref:microsof
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update-qdk).
 
 
+*Release date: May 24th, 2021*
+
+## QDK Version 0.17.2105.xxxxxx
+
+- Updated control-plane clients (.NET, Go, Python) with new swagger. See full details in [Azure REST API specs](https://github.com/Azure/azure-rest-api-specs/pull/13002).
```
Comment (vxfield) [inline articles/qdk-relnotes.md:27]:
```suggestion
```

This item missed the cutoff date and will go into the next release.

**Review by vxfield — CHANGES_REQUESTED** [review]

File: articles/qdk-relnotes.md (hunk 2 above)
Comment (KittyYeungQ) [inline articles/qdk-relnotes.md:40]:
```suggestion
*May 4, 2021*
- Added help button to direct user to support forum.
```

File: articles/qdk-relnotes.md [hunk 9]
```diff
@@ -20,6 +20,27 @@ For installation instructions, please refer to the [install guide](xref:microsof
...
+- Published [QIR oracle generation sample](https://github.com/microsoft/Quantum/issues/496). This program allows turns classical Q# functions on Boolean inputs into quantum implementations of that functions in terms of Q# operations at the level of QIR. This allows, for example, to implement quantum algorithms that are used by many quantum algorithms readily as classical functions.
+- Added Chemistry azure-quantum sample notebook to Quantum repo. 
+- Fixed a bug that prevents QIR generation from being enabled in the iqsharp-base Docker image. See details [here](https://github.com/microsoft/iqsharp/issues/432).
+- Implemented special functions proposal.
+- C# Client: Changed input data format type to "v2" for Quantum Computing. 
+- Set CLI user agent to include version number.
+
+## Azure Quantum update
+
+- Job Scheduler: Fixed an issue where jobs could be rejected if an unrelated metric has reached capacity.
```
Comment (KittyYeungQ) [inline articles/qdk-relnotes.md:38]:
```suggestion

```

File: articles/qdk-relnotes.md (hunk 2 above)
Comment (KittyYeungQ) [inline articles/qdk-relnotes.md:40]:
```suggestion
- Added help button in portal to direct user to support forum.
```

File: articles/qdk-relnotes.md (hunk 3 above)
Comment (KittyYeungQ) [reply articles/qdk-relnotes.md:41]:
```suggestion
- Rendered provider cost in localized currency from Azure Marketplace.
- Added feedback button in portal to gather user feedback.
- Added quickstart guide in portal in overview blade.
```

File: articles/qdk-relnotes.md (hunk 7 above)
Comment (KittyYeungQ) [reply articles/qdk-relnotes.md:36]:
```suggestion
## Azure Quantum update
*May 18th, 2021*

```

File: articles/qdk-relnotes.md (hunk 6 above)
Comment (KittyYeungQ) [reply articles/qdk-relnotes.md:31]: @efratshabtai the ADO did not link to a PR. @cgranade would you mind adding?

Comment (efratshabtai) [reply articles/qdk-relnotes.md:31]: @cgranade

**Review by shenyingjun — APPROVED** [review]

**Review by vxfield — APPROVED** [review]

File: articles/qdk-relnotes.md (hunk 6 above)
Comment (cgranade) [reply articles/qdk-relnotes.md:31]:
```suggestion
- Implemented new special functions (e.g.: factorial and log-gamma) in Microsoft.Quantum.Math (microsoft/QuantumLibraries#448). Thanks to @TheMagicNacho for the contribution (microsoft/QuantumLibraries#440)!
```

File: articles/qdk-relnotes.md [hunk 10]
```diff
@@ -20,6 +20,30 @@ For installation instructions, please refer to the [install guide](xref:microsof
...
+- Added Chemistry azure-quantum sample notebook to Quantum repo. 
+- Fixed a bug that prevents QIR generation from being enabled in the iqsharp-base Docker image. See details [here](https://github.com/microsoft/iqsharp/issues/432).
+- Implemented new special functions (e.g.: factorial and log-gamma) in Microsoft.Quantum.Math (microsoft/QuantumLibraries#448). Thanks to @TheMagicNacho for the contribution (microsoft/QuantumLibraries#440)!
+- C# Client: Changed input data format type to "v2" for Quantum Computing. 
+- Set CLI user agent to include version number.
+
+## Azure Quantum update
+*May 18th, 2021*
+- Added support for new regions: Japan East, Japan West, UK South, UK West
+
```
Comment (KittyYeungQ) [inline articles/qdk-relnotes.md:38]:
```suggestion
```

File: articles/qdk-relnotes.md [hunk 11]
```diff
@@ -20,6 +20,29 @@ For installation instructions, please refer to the [install guide](xref:microsof
...
+- Published [QIR oracle generation sample](https://github.com/microsoft/Quantum/issues/496). This program allows turns classical Q# functions on Boolean inputs into quantum implementations of that functions in terms of Q# operations at the level of QIR. This allows, for example, to implement quantum algorithms that are used by many quantum algorithms readily as classical functions.
+- Added Chemistry azure-quantum sample notebook to Quantum repo. 
+- Fixed a bug that prevents QIR generation from being enabled in the iqsharp-base Docker image. See details [here](https://github.com/microsoft/iqsharp/issues/432).
+- Implemented new special functions (e.g.: factorial and log-gamma) in Microsoft.Quantum.Math (microsoft/QuantumLibraries#448). Thanks to @TheMagicNacho for the contribution (microsoft/QuantumLibraries#440)!
+- C# Client: Changed input data format type to "v2" for Quantum Computing. 
+- Set CLI user agent to include version number.
+
+## Azure Quantum update
```
Comment (KittyYeungQ) [inline articles/qdk-relnotes.md:35]:
```suggestion
## Azure Quantum service update
```

File: articles/qdk-relnotes.md [hunk 12]
```diff
@@ -20,6 +20,29 @@ For installation instructions, please refer to the [install guide](xref:microsof
...
+
+- Added a new function to the `azure-quantum` Python client to support the translation of binary optimization terms from `npz` to Azure Quantum. See full details in [QDK Python](https://github.com/microsoft/qdk-python/pull/61).
+- Published [QIR oracle generation sample](https://github.com/microsoft/Quantum/issues/496). This program allows turns classical Q# functions on Boolean inputs into quantum implementations of that functions in terms of Q# operations at the level of QIR. This allows, for example, to implement quantum algorithms that are used by many quantum algorithms readily as classical functions.
+- Added Chemistry azure-quantum sample notebook to Quantum repo. 
+- Fixed a bug that prevents QIR generation from being enabled in the iqsharp-base Docker image. See details [here](https://github.com/microsoft/iqsharp/issues/432).
+- Implemented new special functions (e.g.: factorial and log-gamma) in Microsoft.Quantum.Math (microsoft/QuantumLibraries#448). Thanks to @TheMagicNacho for the contribution (microsoft/QuantumLibraries#440)!
+- C# Client: Changed input data format type to "v2" for Quantum Computing. 
+- Set CLI user agent to include version number.
```
Comment (ricardo-espinoza) [inline articles/qdk-relnotes.md:33]:
```suggestion
- Released Az CLI quantum extension version 0.5.0: Adapted to 'az' tool version 2.23.0, adding user agent information on calls to Azure Quantum Service.
```

**Review by ricardo-espinoza — APPROVED** [review]

File: articles/qdk-relnotes.md [hunk 13]
```diff
@@ -20,6 +20,29 @@ For installation instructions, please refer to the [install guide](xref:microsof
...
+- Published [QIR oracle generation sample](https://github.com/microsoft/Quantum/issues/496). This program allows turns classical Q# functions on Boolean inputs into quantum implementations of that functions in terms of Q# operations at the level of QIR. This allows, for example, to implement quantum algorithms that are used by many quantum algorithms readily as classical functions.
+- Added Chemistry azure-quantum sample notebook to Quantum repo. 
+- Fixed a bug that prevents QIR generation from being enabled in the iqsharp-base Docker image. See details [here](https://github.com/microsoft/iqsharp/issues/432).
+- Implemented new special functions (e.g.: factorial and log-gamma) in Microsoft.Quantum.Math (microsoft/QuantumLibraries#448). Thanks to @TheMagicNacho for the contribution (microsoft/QuantumLibraries#440)!
+- C# Client: Changed input data format type to "v2" for Quantum Computing. 
+- Released Az CLI quantum extension version 0.5.0: Adapted to 'az' tool version 2.23.0, adding user agent information on calls to Azure Quantum Service.
+
+## Azure Quantum service update
+*May 18th, 2021*
```
Comment (KittyYeungQ) [inline articles/qdk-relnotes.md:36]:
```suggestion
*May 25th, 2021*
- Added PA and SSMC solvers along with preview access via a specialized SKU available to a subset of customers.
*May 18th, 2021*
```

File: articles/qdk-relnotes.md [hunk 14]
```diff
@@ -20,6 +20,31 @@ For installation instructions, please refer to the [install guide](xref:microsof
...
+- Added Chemistry azure-quantum sample notebook to Quantum repo. 
+- Fixed a bug that prevents QIR generation from being enabled in the iqsharp-base Docker image. See details [here](https://github.com/microsoft/iqsharp/issues/432).
+- Implemented new special functions (e.g.: factorial and log-gamma) in Microsoft.Quantum.Math (microsoft/QuantumLibraries#448). Thanks to @TheMagicNacho for the contribution (microsoft/QuantumLibraries#440)!
+- C# Client: Changed input data format type to "v2" for Quantum Computing. 
+- Released Az CLI quantum extension version 0.5.0: Adapted to 'az' tool version 2.23.0, adding user agent information on calls to Azure Quantum Service.
+
+## Azure Quantum service update
+*May 25th, 2021*
+- Added PA and SSMC solvers along with preview access via a specialized SKU available to a subset of customers.
```
Comment (KittyYeungQ) [inline articles/qdk-relnotes.md:37]:
```suggestion
- Added PA (population annealing) and SSMC (sub-stochastic Monte Carlo) solvers along with preview access via a specialized SKU available to a subset of customers.
```

**Comment by ricardo-espinoza** [comment]
Contrary to what GitHub reports, this PR has already been merged. 
See commit `61a23c8adcf44f0a480911d63a6c636de76a20f4`

**Pull request closed without merging**